### ISSUE TITLE
Fails to build with latest unstable

### ISSUE THREAD

**Issue by pimeys** [vlaci/nix-doom-emacs#166]
When updating to nixpkgs unstable, revision `29b0d4d0b600f8f5dd0b86e3362a33d4181938f9`, this flake fails to build:

```
error: --- TypeError --------------------------------------------------------------------------------- nix
at: (69:16) in file: /nix/store/hz2yzvaihcna6gc54mvs4c0mpwv3yqr1-source/lib/customisation.nix

    68|     let
    69|       result = f origArgs;
      |                ^
    70|

anonymous function at /nix/store/7gqxdfavd7mgprp3cnw9pa7iqb41sbjz-source/overrides.nix:1:1 called without required argument 'dune'
(use '--show-trace' to show detailed location information)
```

**Comment by maxhbr** [comment]
`dune` was renamed to `dune_1`, see: https://github.com/NixOS/nixpkgs/commit/60785fe4dbedc4fd2a4a4980c59357a27b57aabf

referenced here in https://github.com/vlaci/nix-doom-emacs/blob/6b0aa3418c574f329d3567dc6e69d955a4908140/overrides.nix#L1

I will try to quickfix it with an overlay that does `dune = super.dune_1`.

**Comment by pimeys** [comment]
Yay. I'll try this overlay after work tonight, thank you!

**Comment by pimeys** [comment]
Fixed for me too: https://github.com/pimeys/nixos/commit/e2c8a336ebdfb6eb3870e016fc2e200e61765a52

**Comment by vlaci** [comment]
Resolved by #165
